steam/guard.py

```diff
@@ -54,7 +54,7 @@
 from time import time
 from steam import webapi
 from steam.enums import ETwoFactorTokenType
-from steam.steamid import SteamID
```

Comment:
That was an unused import.

Reply:
Well, unless documentation for the android `generate_device_id` method actually needs that...

Reply:
Best to leave unrelated changes out of this PR